xml/System.Net.Http/HttpCompletionOption.xml

```diff
@@ -36,7 +36,27 @@
   </Base>
   <Docs>
     <summary>Indicates if <see cref="T:System.Net.Http.HttpClient" /> operations should be considered completed either as soon as a response is available, or after reading the entire response message including the content.</summary>
-    <remarks>To be added.</remarks>
+    <remarks>
+      <format type="text/markdown"><![CDATA[
+
+## Remarks
+ The <xref:HttpCompletionOption> value effectively affects the scope of the timeout specified in the <see cref="T:System.Net.Http.HttpClient" /> operation options when reading a response. The timeout on the <see cref="T:System.Net.Http.HttpClient" /> always applies on the relevant invoked methods up until the point where those methods complete/return. Crucially, when using the <see cref="F:System.Net.Http.HttpCompletionOption.ResponseHeadersRead" /> option, the timeout applies only up to where the headers end and the content starts. The content reading operation needs to be timed out separately in case the server promptly returns the status line and headers but takes too long to return the content. Below is an example illustrating this point:
+
+```csharp
```

Comment:
The .NET contribution docs mention that code snippets have to be supplied in standalone projects in a set folder structure: https://learn.microsoft.com/en-us/contribute/content/dotnet/dotnet-contribute#contribution-process. However, that seems to be for .md docs, not these XMLs (I could be wrong though), and I don't see any similar folder structure in these docs (maybe it's only for https://github.com/dotnet/docs ?), so I'm following the existing pattern (borrowed from HttpClient docs).

Reply:
Examples definitely should be in separate project. Here's example: https://github.com/dotnet/dotnet-api-docs/tree/main/snippets/csharp/System.Reflection.Metadata/MetadataReader

and where it's referenced: https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Reflection.Metadata/MetadataReader.xml#L48

Reply:
Thank you. I've moved the code sample to a standalone snippet along the lines of the MetadataReader example you provided.